build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -283,9 +296,10 @@ function updateEsbuildConfig(config) {
     },
   };
   const babelPlugin = getEsbuildBabelPlugin(
-    'test',
-    /* enableCache */ true,
-    printBabelDot
+    /* callerName */ 'test',
+    /* enableCache */ !argv.watch, // TODO(jridgewell): Make this true when unifiedJsFile goes away.
+    /* preSetup */ logBabelStart,
```

Comment:
FMI: Am I right in thinking `preSetup` fires once per `.build()` call, and `postLoad` is fired once per file?

Reply:
Correct.